dev/index.jsx

```diff
@@ -1,108 +1,44 @@
 import React, { Component } from 'react';
 import ReactDOM from "react-dom";
 import _ from 'lodash';
-import Autosuggest from 'react-autosuggest';
 import MovieDetails from './movieDetails.jsx';
+import MovieSuggestions from './movieSuggestions.jsx';
 
-class Search extends Component {
+class Index extends Component {
```

Comment:
you shouldn't have changed this name. When you are loading this on `app.js` you name it Search, so you can leave it with its name. Index it's not a good name.